DuckDuckGo/SyncSettingsViewController.swift

```diff
@@ -203,6 +201,55 @@ extension SyncSettingsViewController: SyncManagementViewModelDelegate {
         }
     }
 
+    func confirmDisableSync() async -> Bool {
+        return await withCheckedContinuation { continuation in
+            let alert = UIAlertController(title: UserText.syncTurnOffConfirmTitle,
+                                          message: UserText.syncTurnOffConfirmMessage,
+                                          preferredStyle: .alert)
+            alert.addAction(title: UserText.actionCancel) {
```

Comment:
This could use `style: .cancel`, as otherwise the buttons appear in the wrong order:

```suggestion
            alert.addAction(title: UserText.actionCancel, style: .cancel) {
```

Reply:
Thanks!